hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.js

```diff
@@ -0,0 +1,229 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import Button from "@material-ui/core/Button";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import LaunchIcon from "@material-ui/icons/Launch";
+import InputLabel from "@material-ui/core/InputLabel";
+import Chip from "@material-ui/core/Chip";
+import Alert from "@material-ui/lab/Alert";
+
+import styles from "./ProductOverview.module.scss";
+import * as Yup from "yup";
```

Comment:
Minor again, but move the Yup and formik imports above the styles import. The general pattern I like is to have all imports from external libraries up top (some people even alphebetize them, I don't care for that), a line break, then the imports from your own modules. Helps when looking for things.